### DISCUSSION TITLE
Conflicting variable $component, please add reserved keyword to the doc

### DISCUSSION THREAD

**Discussion by RezaODB** [livewire/flux#938] [Feature Requests]
I added a flux:card component to a view and it conflicted with the variable $component that I was using! Please add to the doc all the reserved keywords used by Flux to avoid painful debugging... Thanks!

**Comment by joshhanley** [comment]
@RezaODB `flux:card` doesn't have a `$component` variable in it. Can you share a Volt component that demonstrates the issue (and possibly open it as an issue instead) so we can investigate? Thanks!

**Comment by RezaODB** [comment]
If you use the flux:card component as the single root element in your view, and echo the public $component property in the view it gives you: 
`{"componentName":"flux::card","attributes":[]}`

I will open an issue
